packages/calendar/src/types.ts

```diff
@@ -192,6 +192,16 @@ export type TimeRangeTooltipProps = Omit<TimeRangeDayData, 'date' | 'value'> & {
     value: string
 }
 
+export enum Weekday {
```

Comment:
Unions are simpler to use IMHO, and for non typescript users, we have the same issue as before, it'll be just a number.